src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -200,22 +186,13 @@ const SubmissionDetails = _.flow(
           noContentMessage: 'No matching workflows',
           columns: [
             {
-              size: { basis: 75, grow: 0 },
-              headerRenderer: () => {},
-              cellRenderer: ({ rowIndex }) => {
-                const { workflowId } = filteredWorkflows[rowIndex]
-                return workflowId && h(Link, {
-                  ...Utils.newTabLinkProps,
-                  href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
-                  style: { flexGrow: 1, textAlign: 'center' }
-                }, ['View'])
-              }
-            }, {
-              size: { basis: 225, grow: 0 },
+              size: { basis: 225 },
               headerRenderer: () => h(Sortable, { sort, field: 'workflowEntity', onSort: setSort }, ['Data Entity']),
               cellRenderer: ({ rowIndex }) => {
                 const { workflowEntity: { entityName, entityType } = {} } = filteredWorkflows[rowIndex]
-                return h(TooltipCell, [entityName && `${entityName} (${entityType})`])
+                return h(TooltipCell, [
+                  entityName ? `${entityName} (${entityType})` : span({ style: { color: colors.dark(0.7) } }, ['--'])
+                ])
```

Comment:
I think we don't need the tooltip when there's no entity:

```suggestion
                return !!entityName ?
                  h(TooltipCell, [`${entityName} (${entityType})`]) :
                  div({ style: { color: colors.dark(0.7) } }, ['--'])
```